### ISSUE TITLE
Notification to restart the playback when it's stopped

### ISSUE THREAD

**Issue by pelya** [y20k/transistor#99]
I would like Transistor to show a normal, deletable, non-persistent notification when you stop the playback for any reason, so you can restart the music easily from notification. This notification could disappear in one minute, or it may not disappear at all, it's easy to swipe it away.

The main offender here is Youtube - if I click some Youtube link by mistake, Youtube app will stop Transistor, and there is no way to quickly start it again.
Also few games are grabbing audio focus for no reason when starting  (Pokemon Go, Oddworld), this stops playback, but when you start it again from Transistor's main window, the game will not do this anymore.
Built-in music player on Samsung phones has such notification when stopped, and I think Google Play Music works in the same way.

**Comment by y20k** [comment]
I think your idea is good. You are right: Many media players on my phone do behave this way, too.

Right now you can stop playback using the headphone button and you can re-start playback again using the same button. So a part of the functionality that you are looking for is already there. 

Give me a couple of weeks, I am a bit time constraint and I am still trying to implement [MediaBrowserServiceCompat](https://developer.android.com/reference/android/support/v4/media/MediaBrowserServiceCompat.html) and [MediaControllerCompat](https://developer.android.com/reference/android/support/v4/media/session/MediaControllerCompat.html) correctly. I think this should be done first.

**Comment by pelya** [comment]
Transistor does not fully implement AudioSession interface, so on my phone,
pressing a button once on a Bluetooth headphones stops Transistor, and
pressing it again starts built-in music player, but this is a minor issue I
can live with.

**Comment by y20k** [comment]
> pressing it again starts built-in music player

I my experience that depends also on how the other music players are programmed. On my device some of the players behave as expected, some others are more aggressive in grabbing MediaSessions from other apps. 

Anyway. If possible I would like to fix that problem, too.

**Comment by y20k** [comment]
I did change the way the notification behaves. So far I like it. I think it will be part of the next release. I still need some time to test all the edge cases.

**Comment by pelya** [comment]
Thanks, I will wait for an update.

Another issue is that the song title in the expanded notification is single
line, so it is almost always trimmed, especially during live shows.

There is enough vertical space for two or even three lines, I would set
maxLines property of that text string to 2.

Or I can bring back my scrolling text patch. Would you like scrolling text
in the notification?

**Comment by y20k** [comment]
The new version of Transistor (v2.1.2) includes the improved notification that stays up after playback has been stopped. I will close this issue. Thank you for suggesting it. I really like the outcome. If there are any bugs with it, please open a new issue.

> Would you like scrolling text in the notification?

I am still reluctant to add this feature since non-scolling seems to be the standard for media players

**Comment by pelya** [comment]
New notification works nicely, thank you.

Scrolling text is present in Samsung built-in music player, but not in
Google Music, I did not try other players.
